src/k8s/models.ts

```diff
@@ -68,3 +68,14 @@ export const SecretModel: K8sModel = {
   namespaced: false,
   crd: true,
 };
+
+export const CertManager: K8sModel = {
```

Comment:
Does this work for community cert-manager?

Reply:
Good catch, it doesn't 
![image](https://github.com/user-attachments/assets/5ba1f622-5d92-40de-a783-d7dbcf4a22c6)

I guess I need also to cover that usecase

Reply:
Unfortunately the CRD CertManager doesn't exist when the community cert manager is created.

Reply:
Maybe we can check the presence of the Certificate, Issuer, ClusterIssuer CRDs.